clients/fides-js/src/lib/cookie.ts

```diff
@@ -331,13 +331,21 @@ export const makeConsentDefaultsLegacy = (
 
 /**
  * Given a list of cookies, deletes them from the browser
+ * Optionally removes subdomain cookies as well
  */
-export const removeCookiesFromBrowser = (cookiesToRemove: CookiesType[]) => {
+export const removeCookiesFromBrowser = (
+  cookiesToRemove: CookiesType[],
+  removeSubdomainCookies: boolean = true,
```

Comment:
Do we want this to default to `true` here?

Reply:
yes, this is stored in the DB-level and is unlikely to ever be undefined here. When a user configs their experience in Admin-UI, then that removeSubdomainCookies toggle will be either true or false depending on their choice. 

Additionally, our current customers will get the `false` val written to the DB, so we're safe.